Rubberduck.CodeAnalysis/Inspections/Abstract/QuickFixBase.cs

```diff
@@ -3,6 +3,7 @@
 using System.Diagnostics;
 using System.Linq;
 using NLog;
+using Rubberduck.JunkDrawer.Extensions;
 using Rubberduck.Parsing.Inspections.Abstract;
 using Rubberduck.Parsing.Rewriter;
 using Rubberduck.Parsing.VBA.Extensions;
```

Comment:
If I understand correctly, we moved the `EnumerableExtensions` from `Rubberduck.Parsing.VBA.Extensions` to `Rubberduck.JunkDrawer.Extensions`.

Why isn't this removed? If it can't be removed because it uses other extensions, then why move only one extension and adding yet another `using`?

Same comment applies to several other files that only have the `using` added but not a corresponding removal.

Reply:
I think the simple answer is: because the move refactoring has not removed the now obsolete using.